src/models/CommonModel.ts

```diff
@@ -188,73 +288,16 @@ export class CommonModel extends CommonSchema<CommonModel> {
     }
 
     // Which values are correct to use here? Is allOf required?
```

Comment:
Do not you think that this question should be resolved?

Reply:
I think it is time yea 😄 Gonna add an issue for it.

Reply:
https://github.com/asyncapi/generator-model-sdk/issues/193